Amazon.IonObjectMapper.Test/TestObjects.cs

```diff
@@ -292,10 +292,64 @@ public override string ToString()
 
     public class Wheel
     {
+        public string Specification { get; init; }
+        public int Size { get; init; }
+        public string Brand { get; init; }
+
+        [IonField]
+        internal string color;
+        internal int offset;
+
+        public Wheel(string specification, int size, int offset, string brand, string color)
+        {
+            this.Specification = specification;
+            this.Size = size;
+            this.offset = offset;
+            this.Brand = brand;
+            this.color = color;
+        }
+
         [IonConstructor]
-        public Wheel([IonPropertyName("specification")] string specification)
+        private Wheel(
+            [IonPropertyName("specification")] string specification,
+            [IonPropertyName("size")] int size)
+        {
+            this.Specification = $"Specification: {specification}, Size: {size} inches";
+            this.Size = size;
+        }
+        
+        [IonPropertyGetter("wheel offset")]
+        public int GetOffset() 
         {
+            return this.offset;
+        }
+        
+        [IonPropertySetter("wheel offset")]
+        public void SetOffset(int offset) 
+        {
+            this.offset = offset;
+        }
+    }
 
+    public class Tire
+    {
+        public string specification { get; init; }
+        public int size { get; init; }
+
+        [IonConstructor]
```

Comment:
How does this work? There are no annotated params

Reply:
This `Tire` class is meant for testing the exception when we have multiple `[IonConstructor]`s so the unannotated params did not matter. Regardless I added the param annotations for the sake of completeness.